recipe/meta.yaml

```diff
@@ -11,17 +11,16 @@ source:
         - EHapi.c.patch
 
 build:
-    number: 1
+    number: 2
     skip: True  # [win]
 
 requirements:
     build:
-        - hdf5
+        - hdf5 1.8.15.1
```

Comment:
Indeed. And it sounds like there is a simple solution there... Maybe the problem is the vision for the future: one stable channel/versions solution vs many open possibilities :wink: